operator/k8s_pod_controller.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/cilium/cilium/pkg/controller"
 	"github.com/cilium/cilium/pkg/k8s"
 	slim_corev1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/sirupsen/logrus"
```

Comment:
Nit: we usually keep `github.com/cilium/cilium` and 3rd party imports in separate groups. But not worth re-spinning the PR just to fix this.

Reply:
we can enable local-prefixes settings to check for this, something like below. It might be too much doing now in such large codebase like cilium repo :sweat: 

```
linters-settings:
  goimports:
    local-prefixes: github.com/cilium/cilium
```

Reply:
@sayboras what's the downside?

Reply:
The only downside I see is that there will be a lot of failures for this in the current codebase. Fixing them would mess up the git history for blames. I am not sure what should be the right protocol for such a rework though :thinking: